groups/instance/variables.tf

```diff
@@ -3,17 +3,16 @@ variable "account_name" {
   type        = string
 }
 
+variable "ami_owner_id" {
+  description = "The account ID of the AWS account that owns the AMI"
+  type        = list
+  default     = []
+}
```

Comment:
We usually declare this variable as a string and wouldn't include a default value for this variable.